src/preProcessing/replaceAttributeGeneratingHelpers.ts

```diff
@@ -0,0 +1,48 @@
+import { ReplacementAttributeReference, PreparedTemplate } from './preProcessingTypes';
+import { ATTRIBUTE_GENERATOR_HELPER_FUNCTION } from '../constants';
+import buildNewTemplate from './buildNewTemplate';
+
+const getAllOpeningTagDataRegex = /<([^>]*)\/?>/g;
+
+// NOTE: this regex will identify all explicit helper calls (i.e. a mustache statement with multiple arguments)
+// NOTE: this will not locate helper's that take 0 args since they cannot be distiniguished from basic mustache statements without additional context
```

Comment:
TODO: is there a way to distinguish helpers that take 0 args, perhaps a manual tagging process?